src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java

```diff
@@ -121,6 +128,133 @@ public void configure() {
 
                 });
 
+        /**
+         * Get the process definition key and name
+         *
+         * equivalent es query: GET _search
```

Comment:
Remove this equivalent ES query from here to PR description. Over here just add the request and response.

Reply:
sure